doc/sphinx/Clusters_from_Scratch/active-active.rst

```diff
@@ -27,7 +27,7 @@ cluster filesystems:
 
 .. code-block:: none
 
-    # yum install -y gfs2-utils
+    # dnf install -y gfs2-utils
```

Comment:
Is there a good reason to prefer one or the other? If we were using Fedora I'd definitely go with dnf, but with RHEL all the documentation sticks with yum (last I remember anyway), so it might be worthwhile to stay consistent with that. But if RHEL docs are moving to dnf anytime soon it's probably a good idea here too

Reply:
It doesn't really matter (compatibility is assured for RHEL 9) but the docs have changed. (I **thought** that had already been done in RHEL 8 but nope.) For example:

RHEL 8:
- https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/considerations_in_adopting_rhel_8/software-management_considerations-in-adopting-rhel-8
- https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html-single/managing_software_with_the_dnf_tool/index

RHEL 9:
- https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/considerations_in_adopting_rhel_9/assembly_software-management_considerations-in-adopting-rhel-9
- https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html-single/managing_software_with_the_dnf_tool/index